bin/zero-parachain-live/node/Cargo.toml

```diff
@@ -0,0 +1,101 @@
+[package]
+name = "subzero-parachain-node-live"
+version = "3.1.62"
+authors = ["ZERO <play@zero.io>"]
+description = "A substrate node for video games and beyond."
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+homepage = "https://zero.io"
+repository = "https://github.com/playzero/zero-network/"
+edition = "2021"
+build = "build.rs"
+
+[[bin]]
+name = "subzero-live"
```

Comment:
Binary name for live node `subzero-live`. Are we ok with it? @2075 @vovacha

Reply:
I suggest always assuming parachain and not using this word as a part of any names, even if we will use standalone in the future.

**Template:** {name}-node, {name}-runtime

**Names:**
- `subzero` (main parachain with all the pallets included)
- `subzero-live` (live parachain w/o gamedao protocol pallets)
- `subzero-standalone` (if needed for development)

Reply:
I would go for more intuitive naming conventions that should represent the product and its purpose.
Why not just `zero` instead of `subzero`? Do we have some product/feature/module called **subzero**?
Also `live` is not obvious considering there are no protocol pallets. I would suggest `base` or `alfa` or `ground`.
So, something like this:
- zero
- zero-base
- zero-standalone

Reply:
> I would go for more intuitive naming conventions that should represent the product and its purpose. Why not just `zero` instead of `subzero`?

Actually, I am also not sure why we are using `subzero` instead of `zero`. Probably @2075 already explained the reasons, but I forgot what was that.
If it's a zero network, doesn't it make sense to use the same name for the runtime?

> I would suggest base or alfa or ground.

- `base` or `empty` sounds like there is nothing there, and it's true, but maybe we don't want to expose it through the name.
- `ground` - doesn't like it.
- `alfa` - maybe. But also, `live` is quite descriptive.

Reply:
And `subzero-standalone` for standalone chain. Sounds good.